app/src/Components/Container/Container.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
```

Comment:
[?] Do you really understand what's going on here? If not, I would suggest re-writing your edit UX using something that you *do* understand.

Reply:
For example, you can consider something like this if you want to keep the contact list in a tabular form: https://codesandbox.io/s/tabular-form-4cuxi